docs/platform-feature-parity.md

```diff
@@ -0,0 +1,171 @@
+# Platform feature-parity
```

Comment:
Need some small note to set expectations on multi-os "community support".

Basically, that I make no promises of support or timeline on adding features/fixing bugs for those platforms.

Realistically, I probably will dabble in some bugfixes/features. I'd just rather do that optionally rather than as an expectation.

Reply:
Done in 17d2a39, please take a look?